code/core/src/telemetry/storybook-metadata.ts

```diff
@@ -77,7 +77,7 @@ export const computeStorybookMetadata = async ({
     metadata.metaFramework = {
       name: metaFrameworks[metaFramework],
       packageName: metaFramework,
-      version,
+      version: version || 'unknown',
```

Comment:
why `undefined` elsewhere and `'unknown'` here?

Reply:
The type used to be `any`.

I made it more strict, but that didn't match up with the expected type here.
I just made it work without too much consideration.

I can also type-cast it back to `any` if you feel that's better.
I didn't want to keep pulling the thread and potentially run out of time.